controllers/articles.controller.js

```diff
@@ -16,11 +17,25 @@ exports.getArticlesById = (request, response, next) => {
 };
 
 exports.getArticles = (request, response, next) => {
-  selectArticles()
-    .then((articles) => {
-      response.status(200).send({ articles });
-    })
-    .catch(next);
+  const { topic, sort_by, order } = request.query;
+  if (topic) {
+    const promises = [
+      selectArticleQueries(topic, sort_by, order),
```

Comment:
I feel these should be in reverse order, but still, if one rejects I suppose it doesn't really matter.